src/errors.js

```diff
@@ -9,10 +9,13 @@ export class NotImplementedError extends Error {
 
 export class HttpError extends Error {
   constructor(message, status) {
-    super(message);
-
-    this.message = message;
-    this.status = status;
-    this.name = 'HttpError';
+    super(message.message);
```

Comment:
totally optional, but naming looks a little bit awkward, I rename the parameter to `response` oslt